ckanext/short_urls/model.py

```diff
@@ -0,0 +1,50 @@
+import string
+import random
+from sqlalchemy import Column, types, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from ckan.model.meta import metadata
+
+Base = declarative_base(metadata=metadata)
+
+OBJECT_TYPE_DATASET = 'dataset'
+OBJECT_TYPE_RESOURCE = 'resource'
+
+
+def _generate_random_string(string_length=6):
```

Comment:
Let's make default length `8` to make sure we're avoding collisions.
Also rename the param to just `length`.